_posts/2023-07-05-lts-releases.adoc

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: 'Long-Term Support(LTS) for Quarkus'
+date: 2023-07-05
+tags: announcements
+synopsis: 'We are introducing Quarkus Long Term Support releases, Quarkus 3.2.x will be the first.'
+author: maxandersen
+---
+
+We're thrilled to announce the introduction of Long-Term Support (LTS) releases for Quarkus. We aim to strike a balance between our regular high-paced release cycle and the needs of users who require more stability and predictability.
+
+LTS releases offer an opportunity for our community to enjoy a version of Quarkus, with the added assurance of critical patches, bug fixes, and security updates. We understand that not every user wants to continuously upgrade to the latest release and some prefer to stick with a version they know works for them. Until now Red Hat Build of Quarkus provided such option, but we want to ensure Quarkus community project and especially community projects built on top of Quarkus has similar options.
+
+## Quarkus Users
+
+Here's what you need to know as a Quarkus user:
+
+. *Flexibility*: We still recommend users to upgrade early and often, but users now have the choice to upgrade to the latest feature-packed version or stick with the stability and predictability of an LTS release. This decision can be made based on individual requirements and risk tolerance.
+
+. *Stability*: LTS versions will be supported for a period of 12 months, allowing users to maintain the same version of Quarkus without worrying about new features introducing regressions or breaking changes. We aim to release a new LTS version every 6 months.
+
+. *Long-Term Support*: LTS versions will continue to receive critical bug fixes, security patches, and performance enhancements even after the introduction of newer versions. Critical issues that directly impact functionality, performance, or security will be prioritized for fixes.
+
+. *Security*: Regular security updates will be part of LTS releases, ensuring your applications stay protected against known vulnerabilities.
+
+. *Identification*: LTS releases will for now be identifed on https://quarkus.io/security. We are working on adding a new badge to the Quarkus website to make it easier to identify LTS releases.
+
+. *Implications for Non-LTS Releases*: Our regular monthly'ish release cycle will continue as before, providing feature updates and improvements at a high pace for those who prefer to stay on the cutting edge.
+
+## Quarkus Extension Maintainers and Contributors
+
+All of the above applies to Quarkus extension maintainers and contributors as well. In addition, we will be recommending that extension maintainers and contributors considers bug fixes and enhancements for LTS releases. This will ensure that LTS releases are as stable and robust as possible.
```

Comment:
```suggestion
All of the above applies to Quarkus extension maintainers and contributors as well. In addition, we will be recommending that extension maintainers and contributors consider bug fixes and enhancements for LTS releases. This will ensure that LTS releases are as stable and robust as possible.
```